From: Benjamin Marzinski @ 2025-02-11 18:11 UTC (permalink / raw)
  To: Christophe Varoqui; +Cc: device-mapper development, Martin Wilck

urcu/atomic.h requires some header files included by urcu.h. Make sure
to include it first.

Signed-off-by: Benjamin Marzinski <bmarzins@redhat.com>
---
 libmultipath/checkers/tur.c | 1 +
 libmultipath/lock.h         | 1 +
 2 files changed, 2 insertions(+)

diff --git a/libmultipath/checkers/tur.c b/libmultipath/checkers/tur.c
index e70a2e11..0010acf8 100644
--- a/libmultipath/checkers/tur.c
+++ b/libmultipath/checkers/tur.c
@@ -15,6 +15,7 @@
 #include <errno.h>
 #include <sys/time.h>
 #include <pthread.h>
+#include <urcu.h>
 #include <urcu/uatomic.h>
 
 #include "checkers.h"
diff --git a/libmultipath/lock.h b/libmultipath/lock.h
index 38473a8c..5f323055 100644
--- a/libmultipath/lock.h
+++ b/libmultipath/lock.h
@@ -2,6 +2,7 @@
 #define LOCK_H_INCLUDED
 
 #include <pthread.h>
+#include <urcu.h>
 #include <urcu/uatomic.h>
 #include <stdbool.h>
 
-- 
2.48.1
